apiCallbacks/get-mms.md

```diff
@@ -30,7 +31,8 @@
 	applicationId=a-yr3jpxasdfh5xh5e35saoi&
 	direction=in&
 	media=[{full-url}/giphy-m-asdf.gif, {full-url}/123_1-m-asdf.smil]
-	messageUri=https%3A%2F%2Fapi.catapult.inetwork.com%2Fv1%2Fusers%2Fu-123%2Fmessages%2Fm-asdf
+	messageUri=https%3A%2F%2Fapi.catapult.inetwork.com%2Fv1%2Fusers%2Fu-123%2Fmessages%2Fm-asdf&
+    segmentCount=1
```

Comment:
Not sure this should be added here.

Reply:
This is for callbacks that were sent via GET. If this isn't supposed to be there then would the callbacks sent via POST also not have this?

```
  "media"               : ["mediaUri1", "mediaUri2"...],
  "segmentCount":       : integer
}
```

Reply:
Right, but the message URI is one of the fields passed via get, so it shouldnt have segmentCount duplicated within it. Just test this with a GET callback and the messageURI was "https://api.catapult.inetwork.com/v1/users/u-ID/messages/m-ID"

Reply:
Oh. I see. But the `segmentCount` is not part of the `messageURI`. It's a whole separate field

Reply:
It's like `direction` or `applicationId` unless I'm mistaken

Reply:
Yes, the segmentCount is not part of the messageURI, the messageURI is a separate piece of data returned in the callback.